inductiva/structures/post_processing/fenicsx.py

```diff
@@ -0,0 +1,149 @@
+"""Post-process FEniCSx simulation outputs."""
+
+import os
+import tempfile
+
+import pyvista as pv
+
+import inductiva
+
+
+class FEniCSxSimulationOutput:
+    """Post-process FEniCSx simulation outputs."""
+
+    def __init__(self, sim_output_path: inductiva.types.Path):
+        """Initializes a 'FEniCSxSimulationOutput' object.
+
+        Args:
+            sim_output_path: Path to simulation output files.
+            """
+        self.sim_output_path = sim_output_path
+
+    def remove_time_values_from_xdmf(self, output_file_path: str) -> None:
```

Comment:
Isn't this function an utility function? It maybe useful for other future scenarios.

Reply:
Yes, you're correct, I'll move it to the utils of the structure scenario, thanks!